src/arrayMethodReduce.js

```diff
@@ -5,7 +5,14 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let previous = (arguments.length < 2) ? this[0] : initialValue;
+    const startIndex = (arguments.length < 2) ? 1 : 0;
```

Comment:
ternary expression should be multiline
```suggestion
    let previous = (arguments.length < 2) 
      ? this[0] 
      : initialValue;
    const startIndex = (arguments.length < 2) ? 1 : 0;
```

Reply:
also you can create one if statement where you will do this instead of creating two ternary expression